ios/ViewController.mm

```diff
@@ -144,7 +145,8 @@ - (void)subtleVolume:(SubtleVolume *)volumeView didChange:(CGFloat)value {
 - (void)viewDidLoad {
 	[super viewDidLoad];
 
-	self.view.frame = [[UIScreen mainScreen] bounds];
+    UIScreen* screen = [(AppDelegate*)[UIApplication sharedApplication].delegate screen];
```

Comment:
It'd be good to keep consistent with tabs throughout.  In PPSSPP, we use tabs so we just want to keep it consistent.

-[Unknown]

Reply:
Xcode use spaces for indentation by default. I have changed it to tabs.